src/vue/forms/DepositForm.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="deposit">
+    <template v-if="isLoaded">
+      <template v-if="form.assetCode">
+        <div class="app__form-row deposit__form-row">
```

Comment:
it's outside from the `form`. why it's called `form-row`?

Reply:
ok